wolfcrypt/test/test.c

```diff
@@ -18916,6 +18981,64 @@ int ed25519_test(void)
 #endif /* HAVE_ED25519_VERIFY */
     }
 
+    outlen = sizeof(out);
+    XMEMSET(out, 0, sizeof(out));
+    
+    if (wc_ed25519_import_private_key(sKeyCtx1, ED25519_KEY_SIZE, pKeyCtx1,
+                                      sizeof(pKeyCtx1), &key) != 0)
+        return -9020;
+
+    if (wc_ed25519ctx_sign_msg(msgCtx1, sizeof(msgCtx1), out, &outlen, &key,
```

Comment:
Can we add a test for ctx_sign_msg with no context?

Reply:
Added - no test vector in document